### PR TITLE
Add Tests for FX-to-ONNX exporter

### PR DESCRIPTION
**Pull request by wschin**
Run some tests from OpInfo to test FX-to-ONNX exporter.

### CODE REVIEW

File: test/onnx/test_fx_to_onnx_with_onnxruntime_for_every_op.py [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+
+                try:
+                    gm = functorch.make_fx(op)(*copy.deepcopy(args))
+                    self.run_test_with_fx_to_onnx_exporter(
+                        gm, args, rtol=1e-3, atol=1e-7
+                    )
+                except Exception as e:
+                    failed_op_names.add(op.name)
+
+        self.assertEqual(len(failed_op_names), 0, f"Failed op names: {failed_op_names}")
```
Comment (justinchuby) [inline test/onnx/test_fx_to_onnx_with_onnxruntime_for_every_op.py:223]: I would parameterize the tests. Let me know if you need examples.

Comment (wschin) [reply test/onnx/test_fx_to_onnx_with_onnxruntime_for_every_op.py:223]: Sure. Please give me some examples I can follow. Thanks.

Comment (justinchuby) [reply test/onnx/test_fx_to_onnx_with_onnxruntime_for_every_op.py:223]: See if this one helps? https://github.com/microsoft/onnx-script/pull/208/files#diff-db423ee0dcd2d4a7c3dddb5141d8b3e2b4cc3f60b32354e36ec14ac3f61dbc01

You would simply need to replace the op name mapping with a list

**Comment by titaiwangms** [comment]
I tried put fx_export test function `self.run_test_with_fx_to_onnx_exporter()` into `test_op_consistency.py` to avoid a new test file, but it doesn't work, as dynamo not yet supports call function from user defined object (IIUC op parameter in SingleOpModel is a user defined object).

**Pull request closed without merging**